Tests/SpeziFoundationTests/ExceptionHandlingTests.swift

```diff
@@ -10,6 +10,7 @@
 import SpeziFoundation
 import XCTest
 
+#if !os(Linux) // Objective C not supported in Linux, since there is no Objective C runtime
```

Comment:
Similar comment as noted above:
```suggestion
#if canImport(ObjectiveC)
```